src/stylesheets/main.scss

```diff
@@ -69,14 +69,9 @@ $mq-breakpoint-widescreen: 1200px;
   }
 }
 
-// Note: on Elements this is included on the body - we will review in FE how to best add font smoothing
-.govuk-u-font-smoothing {
-  @include font-smoothing;
-}
-
 .app-c-content {
-  padding: $govuk-gutter;
-  @include govuk-core-19;
+  padding: $govuk-gutter-half;
```

Comment:
I don't think this is correct – should this be using responsive padding? At the minute the content within the right hand pane only has 15px around it on a tablet breakpoint, but in current master it's 30px.

Reply:
(I appreciate I might have made this change!)

Reply:
Nope, this was my change 😄